plugins/inputs/ceph/README.md

```diff
@@ -168,32 +168,52 @@ All admin measurements will have the following tags:
 
 ## Cluster
 
+- ceph_fsmap
+  - fields:
+    - up (float)
+    - in (float)
+    - max (float)
+    - up_standby (float)
+
 - ceph_health
   - fields:
-    - status
-    - overall_status
+    - status (string)
+    - status_code (int)
+
+- ceph_monmap
+  - fields:
+    - num_mons (float)
 
 - ceph_osdmap
   - fields:
     - epoch (float)
     - num_osds (float)
     - num_up_osds (float)
     - num_in_osds (float)
-    - full (bool)
-    - nearfull (bool)
```

Comment:
Similar for these. You could add info if they only existed before a specific ceph version..

Reply:
Re-added with latest updates